README.md

```diff
@@ -26,54 +26,56 @@ $ mv kperf /usr/local/bin/
 
 Note: [go-bindata](https://github.com/go-bindata/go-bindata) is required in the build process.
 
-## User Stories
+## Knative Serving load test
 
-### As a Knative developer, I want to generate Knative Service concurrently for test
+Kperf can help to generate Knative Service Deployment Load in your Knative platform. We assume you have created a
+Kubernetes cluster and deployed [Knative Serving](https://knative.dev/docs/install/). 
+
+Please note that by default kperf assumes you have prepared K8s namespace(s) to create Knative Service. 
+If namespace doesn't exist, create it with kubectl as below
+
+```shell script
+# Create a namespace for kperf to create Services in it
+kubectl create ns {namespace-name}
+
+# Create namespaces from test-1 to test-10 for kperf to create Services distributed in them
+for name in {1..10};do kubectl create ns test-$name;done
 ```
+
+### generate Knative Service deployment load
+```shell script
 # Generate total 30 knative service, for each 15 seconds create 10 ksvc with 5 concurrency in namespace test1, test2 and test3, and the ksvc names are ktest1, ktest2.....ktest29.
 $ kperf service generate -n 30 -b 10 -c 5 -i 15 --nsPrefix test --nsRange 1,3 --svcPrefix ktest --maxScale 1 --minScale 1
```

Comment:
According to the latest code:
`nsPrefix` => `namespace-prefix`
`nsRange` => `namespace-range`
`svcPrefix ` => `svc-prefix`
`minScale ` => `min-scale`
`maxScale ` => `max-scale`

Reply:
will cover by a rebase.